includes/language/japanese.php

```diff
@@ -52,7 +54,7 @@
     'admin_action_pw_prefix_correct_tip' => 'Please backup the database before performing this task. This script will perform an update of password prefix in database. Only run this to fix incorrectly displayed passwords after upgrading from older versions.',
     'items_changed' => '変更しました。',
     'ga_not_yet_synchronized' => 'Get identified with Google Authenticator',
-    'ga_scan_url' => 'Google認証アプリなどでこのQRコードをスキャンしてください�',
+    'ga_scan_url' => 'Google隱崎ｨｼ繧｢繝励Μ縺ｪ縺ｩ縺ｧ縺薙�QR繧ｳ繝ｼ繝峨ｒ繧ｹ繧ｭ繝｣繝ｳ縺励※縺上□縺輔＞',
```

Comment:
As best as I can tell, this Unicode value was not changed by my code, but still shows as different, I have updated the value to the exact same string in GitHub directly, however, this difference still shows.